spec/system/relationships_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Relationships', type: :system do
+  let!(:user) { create(:user) }
+  let!(:friend) { create(:user) }
+  let!(:post) { create(:post, user: friend) }
+
+  before do
+    sign_in user
+  end
+
+  it 'フォローできること' do
+    visit root_path
+    expect do
+      click_link 'フォローする'
+    end.to change(user.following, :count).by(1)
+    expect(page).to have_content('フォロー中')
+  end
+
+  it 'フォローを外せること' do
+    user.follow(friend)
+    visit root_path
+    expect do
+      click_button 'フォロー中'
```

Comment:
フォロー中は「ステータス」の表示で、このラベルを押すのがイメージできなかった。
これでどうだろう？
```suggestion
      click_button 'フォロー解除'
```

Reply:
twitterを真似たつもりだったのですが、twitterはfollowingをホバーするとunfollowテキストに変化する仕様になっていました。
そうします！